setup.py

```diff
@@ -60,5 +60,5 @@ class CustomInstall(install):
         "console_scripts": ["check50=check50:main"]
     },
     url="https://github.com/cs50/check50",
-    version="2.0.6"
+    version="2.0.8"
```

Comment:
Why `8`?

Reply:
Actually, let's wait before merging, couple other changes pending it seems. And we should go to 2.1.0 anyway.

Reply:
:+1: